recipes/protobuf/all/conanfile.py

```diff
@@ -73,67 +84,75 @@ def _configure_cmake(self):
             self._cmake.definitions["protobuf_WITH_ZLIB"] = self.options.with_zlib
             self._cmake.definitions["protobuf_BUILD_TESTS"] = False
             self._cmake.definitions["protobuf_BUILD_PROTOC_BINARIES"] = True
-            self._cmake.definitions["protobuf_BUILD_LIBPROTOC"] = True
-            if self.settings.compiler == "Visual Studio":
-                self._cmake.definitions["protobuf_MSVC_STATIC_RUNTIME"] = "MT" in str(self.settings.compiler.runtime)
+            if tools.Version(self.version) >= "3.14.0":
+                self._cmake.definitions["protobuf_BUILD_LIBPROTOC"] = True
+            if tools.Version(self.version) >= "3.15.4":
+                self._cmake.definitions["protobuf_DISABLE_RTTI"] = False # TODO: create an option for 3.15.4+?
```

Comment:
❗ https://github.com/conan-io/conan-center-index/pull/4682#discussion_r587947898

I think this release/feature might be broken, see the convo above

Reply:
@floriansimon1 Is the usage here a problem?

Reply:
Pretty much, yes. @SpaceIm, if we merge this, it means we won't be able to disable RTTI, which is a feature I explicitly need for 3.15.5. Why disable it?

Reply:
Like the TODO says, add an option? 😃
@SpaceIm, that is what @floriansimon1 PR is about  https://github.com/conan-io/conan-center-index/pull/4682/

Reply:
Yes, I saw the content of this PR after adding 3.15.5 in this one. I guess https://github.com/conan-io/conan-center-index/pull/4776/commits/3a565443a21718d508a96757d95894c588a2461b solves the problem, but maybe the best would be to move forward on #4682 and merge it.